data/styles/clear/include/Subways.mapcss

```diff
@@ -95,11 +95,13 @@ node|z17-[railway=station][transport=subway][city=berlin] {icon-image: subway-ub
 node|z16-[railway=subway_entrance][city=berlin] {icon-image: subway-ubahn-s.svg;}
 
 /* Boston Subway Station*/
+/* TODO: add a mapping and a city bbox if missing
```

Comment:
@RedAuburn could you please check why don't we have Boston and Lisbon subways in the mapping? (when you have time)